packages/tiered/src/index.ts

```diff
@@ -0,0 +1,119 @@
+import EventEmitter from 'node:events';
+import Keyv from 'keyv';
+import type {Options, Options_} from './types';
+
+type KeyvTieredIndex = 'local' | 'remote';
+
+class KeyvTiered extends EventEmitter {
+	opts: Options_;
+	remote: Keyv;
+	local: Keyv;
+	iterationLimit?: string | number;
+
+	constructor({remote = new Keyv(), local = new Keyv(), ...options}: Options) {
+		super();
+		this.opts = {
+			validator: () => true,
+			dialect: 'tiered',
+			...options,
+		};
+		this.remote = remote;
+		this.local = local;
+	}
+
+	async get(key: string): Promise<any> {
```

Comment:
Do we want to return `any` or could we define this more?

Reply:
We don't know the data type cached, for this reason, we can return any.